roles/container_runtime/tasks/package_crio.yml

```diff
@@ -46,11 +46,19 @@
     - /etc/cni/net.d/200-loopback.conf
     - /etc/cni/net.d/100-crio-bridge.conf
 
-- name: Create the CRI-O configuration
-  template:
+- name: Set pause_image to pod image
+  ini_file:
+    dest: /etc/crio/crio.conf
+    section: "crio.image"
+    option: pause_image
+    value: '"{{ openshift_crio_pause_image }}"'
+
+- name: Set pause_command to pod cmd
+  ini_file:
     dest: /etc/crio/crio.conf
-    src: crio.conf.j2
-    backup: yes
+    section: "crio.image"
+    option: pause_command
+    value: ' "/usr/bin/pod"'
```

Comment:
you didn't like the template ?

imo a template would give you a much better flexibility in the long run when things will get much more stable and you will realize you want to do some tuning.

Reply:
We don't directly control the crio versions, unlike docker, so its best to modify just the required parts

Reply:
right, i was coming from the angle where later the default CRI-O settings won't satisfy the needs of BYO .. i guess we can cross that bridge when we come to it